app/utils/Format.scala

```diff
@@ -31,4 +31,7 @@ object Format {
   def timeFormatted(time: LocalTime): String         = time.format(timeFormatter)
 
   val dateTimeFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMddHHmm")
+
+  val isoDateFormatter: DateTimeFormatter     = DateTimeFormatter.ISO_LOCAL_DATE
```

Comment:
isoDateFormatter sets the same val twice...is it needed?

Reply:
Does it? One has hyphens the other doesn't!

Reply:
isoDateFormatter is set to DateTimeFormatter.ISO_LOCAL_DATE, I was just wondering if we need the isoDateFormatter when we can just use DateTimeFormatter.ISO_LOCAL_DATE?

Reply:
This might be a good subject for a team discussion and documented agreement. As long as the date/time format is ISO 8601 compliant, maybe we could have a single library used by everyone to format date/times. Personally I like to see the zoneId rather than hourly offset, because offsets can be a bit misleading.

Reply:
Agree - we should discuss and have a single source for time/date.